index.html

```diff
@@ -11821,7 +11821,8 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>If the interface is such that it is not possible to have a visible label on the screen, authors SHOULD use <pref>aria-label</pref> and SHOULD NOT use <pref>aria-labelledby</pref>. Authors MUST NOT specify <code>aria-labelledby</code> on an element which has an explicit or implicit WAI-ARIA role where <code>aria-labelledby</code> is <a href="#prohibitedattributes">prohibited</a>. As required by the <a href="#textalternativecomputation">accessible name and description computation</a>, user agents give precedence to <pref>aria-labelledby</pref> over <pref>aria-label</pref> when computing the accessible name property.</p>
 				<p>The <pref>aria-labelledby</pref> attribute is similar to <pref>aria-describedby</pref> in that both reference other elements to calculate a text alternative (an accessible name, and description, respectively). While a concise accessible name is preferable, a description can either be concise, or provide more verbose information.</p>
 				<!-- keep previous sentence synced with the associated description in #aria-describedby -->
-				<p class="note">The expected spelling of this property in <abbr title="United States">U.S.</abbr> English is "labeledby." However, the <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> features to which this property is mapped have established the "labelledby" spelling. This property is spelled that way to match the convention and minimize the difficulty for developers.</p>
+				<p class="note">The expected spelling of this property in <abbr title="United States">U.S.</abbr> English is "labeled by." However, the <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> feature this property was originally based on had established the <abbr title="United Kingdom">U.K.</abbr> English spelling, "labelled by."</p>
+				<p>To mitigate against common author use of the "labeledby" misspelling, thus ensuring more content will be properly named, user agents MUST support the canonical "aria-labelledby" and  MAY support the alternate "aria-labeledby" spelling when the canonical variant is missing. Authors MUST NOT use the alternate "aria-labeledby" variant of the attribute. For instances where an author has erroneously used both spelling variants on the same element, user agents MUST use the canonical <code>aria-labelledby</code> spelling and ignore the alternate spelling of the attribute.</p>
```

Comment:
```suggestion
				<p>To mitigate against common author use of the "labeledby" misspelling, thus ensuring more content will be properly named, user agents MUST support the canonical "aria-labelledby" and  MAY support the alternate "aria-labeledby" spelling when the canonical variant is missing. Authors SHOULD NOT use the alternate "aria-labeledby" variant of the attribute. For instances where an author has erroneously used both spelling variants on the same element, user agents MUST use the canonical <code>aria-labelledby</code> spelling and ignore the alternate spelling of the attribute.</p>
```

Reply:
I was originally saying MUST NOT to use of both spellings of the attribute together.  I personally would rather this rewording be a SHOULD NOT than a MUST NOT @cookiecrook 

the reason being that if it properly works to name the element, then i wouldn't want an a11y checker telling an author they did something wrong.  but a code linter could still point out this spelling error in a way that doesn't imply the author has made an error that would impact the accessibility of the element being named.

Reply:
> I was originally saying MUST NOT to use of both spellings of the attribute together. I personally would rather this rewording be a SHOULD NOT than a MUST NOT @cookiecrook

So do you want to pull the original "together" back in with the alternate you merged? like this?

```suggestion
				<p>To mitigate against common author use of the "labeledby" misspelling, thus ensuring more content will be properly named, user agents MUST support the canonical "aria-labelledby" and  MAY support the alternate "aria-labeledby" spelling when the canonical variant is missing. Authors MUST NOT use both spelling variants on the same element and SHOULD NOT use the alternate "aria-labeledby" variant of the attribute. For instances where an author has erroneously used both spelling variants on the same element, user agents MUST use the canonical <code>aria-labelledby</code> spelling and ignore the alternate spelling of the attribute.</p>
```

Or do you want to just change the MUST NOT to a SHOULD NOT? I think I am fine with either outcome.

Reply:
i'll pull in what you revised this to.  i'm good with that - gives conformance checkers something to flag an error about (using both together is a mistake and it could lead to the wrong name being assigned), and then the lesser warning to use the proper spelling